data/mods/gen3/random-teams.ts

```diff
@@ -389,162 +472,127 @@ export class RandomGen3Teams extends RandomGen4Teams {
-					!(types.has('Ghost') && species.baseStats.spa > species.baseStats.atk) &&
-					!(
-						// With Calm Mind, Lugia and pure Normal-types are fine without STAB
-						counter.setupType === 'Special' && (
-							species.id === 'lugia' ||
-							(types.has('Normal') && species.types.length < 2)
-						)
-					) &&
-					!(
-						// With Swords Dance, Dark-types and pure Water-types are fine without STAB
-						counter.setupType === 'Physical' &&
-						((types.has('Water') && species.types.length < 2) || types.has('Dark'))
-					) &&
-					counter.get('physicalpool') + counter.get('specialpool') > 0
-				);
-
-				const runEnforcementChecker = (checkerName: string) => {
-					if (!this.moveEnforcementCheckers[checkerName]) return false;
-					return this.moveEnforcementCheckers[checkerName](
-						movePool, moves, abilities, types, counter, species as Species, teamDetails
-					);
-				};
-
-				if (!cull && !isSetup && moveIsRejectable) {
-					// There may be more important moves that this Pokemon needs
-					if (
-						requiresStab ||
-						(counter.setupType && counter.get(counter.setupType) < 2 && !moves.has('refresh')) ||
-						(moves.has('substitute') && movePool.includes('morningsun')) ||
-						['meteormash', 'spore', 'recover'].some(m => movePool.includes(m))
-					) {
-						cull = true;
-					} else {
-						// Pokemon should have moves that benefit their typing and their other moves
-						for (const type of types) {
-							if (runEnforcementChecker(type)) {
-								cull = true;
-							}
-						}
-						for (const m of moves) {
-							if (runEnforcementChecker(m)) cull = true;
-						}
-					}
-				}
+		let ability = '';
+		let item = undefined;
 
-				// Sleep Talk shouldn't be selected without Rest
-				if (moveid === 'rest' && cull) {
-					const sleeptalk = movePool.indexOf('sleeptalk');
-					if (sleeptalk >= 0) {
-						if (movePool.length < 2) {
-							cull = false;
-						} else {
-							this.fastPop(movePool, sleeptalk);
-						}
-					}
-				}
+		const evs = {hp: 85, atk: 85, def: 85, spa: 85, spd: 85, spe: 85};
+		const ivs = {hp: 31, atk: 31, def: 31, spa: 31, spd: 31, spe: 31};
 
-				// Remove rejected moves from the move list
-				const moveIsHP = moveid.startsWith('hiddenpower');
-				if (
-					cull &&
-					(movePool.length - availableHP || availableHP && (moveIsHP || !hasHiddenPower))
-				) {
-					if (move.category !== 'Status' && !move.damage && (!moveIsHP || !availableHP)) {
-						rejectedPool.push(moveid);
-					}
-					if (moveIsHP) hasHiddenPower = false;
-					moves.delete(moveid);
-					break;
-				}
-				if (cull && rejectedPool.length) {
-					if (moveIsHP) hasHiddenPower = false;
-					moves.delete(moveid);
-					break;
-				}
-			}
-		} while (moves.size < this.maxMoveCount && (movePool.length || rejectedPool.length));
+		const types = species.types;
+		const abilities = new Set(Object.values(species.abilities));
+		if (species.unreleasedHidden) abilities.delete(species.abilities.H);
```

Comment:
```suggestion
		const abilities = new Set(Object.values(species.abilities));
```
HAs dont exist in gen 3